app/src/main/java/com/daotrung/listcountry/model/Country.kt

```diff
@@ -0,0 +1,9 @@
+package com.daotrung.listcountry.model
+
+data class Country(
+    val id: Int,
+    val nameCountry: String,
+    val populationCountry: String,
```

Comment:
```suggestion
    val population: String,
```

Population sao lại là string mà k phải long?